src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -140,7 +141,8 @@ public function dump(array $options = array())
         ;
         $this->targetDirRegex = null;
 
-        return $code;
+        // Replace '/**' comment openings with '/*' for non-debug container to optimise for opcache
```

Comment:
`optimise` -> `optimize` (we mostly use American English in code and docs) Thanks!